R/init.R

```diff
@@ -237,13 +241,3 @@ init <- function(data,
 
   return(res)
 }
-
-create_app_id <- function(data, modules) {
-  hashables <- c(data, modules)
-  hashables$data <- if (inherits(hashables$data, "teal_data")) {
-    as.list(hashables$data@env)
-  } else if (inherits(data, "teal_data_module")) {
-    body(data$server)
-  }
-  rlang::hash(hashables)
-}
```

Comment:
I don't see this block on `main` :thinking:

Reply:
IDK why it's highlighted in `R/init.R` as removed :/

I moved respective code to the function and placed it in the utils script.